src/convertToObject.js

```diff
@@ -3,16 +3,37 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string
+ * with styles (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related
+ *  CSS properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .trim()
+    .replace(/;/g, '')
+    .split('\n');
+
+  const convertStyle = styles.reduce((obj, el) => {
+    if (el.trim() === '') {
+      return obj;
+    }
```

Comment:
```suggestion
    if (!el.trim()) {
      return obj;
    }
```
You don't need to compare with an empty string, just check if a value exists
https://developer.mozilla.org/en-US/docs/Glossary/Falsy